### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,37 +7,37 @@
 
 Lance is a cloud-native columnar data format designed for managing large-scale computer vision datasets in production
 environments. Lance delivers blazing fast performance for image and video data use cases from analytics to point 
-queries to training scans. 
+queries to training scans.
 
-## Why use Lance
+Lance core is written in C++ and comes with python bindings to start. With first class integration, Lance is queryable by tools like DuckDB out of the box and can be converted from parquet with a single line of code.
```
Comment (eddyxu) [inline README.md:12]: `first class Apache Arrow integration`?

**Review by eddyxu — APPROVED** [review]
Small nits

File: README.md (hunk 1 above)
Comment (changhiskhan) [reply README.md:12]: Yes. typo from original doc

**Comment by eddyxu** [comment]
lance_perf

**Pull request merged**